packages/edit-post/src/components/keyboard-shortcut-help-modal/index.js

```diff
@@ -88,14 +67,45 @@ export function KeyboardShortcutHelpModal( { isModalActive, toggleModal } ) {
 			/>
 			{ isModalActive && (
 				<Modal
-					className="edit-post-keyboard-shortcut-help"
+					className="edit-post-keyboard-shortcut-help-modal"
 					title={ __( 'Keyboard shortcuts' ) }
 					closeLabel={ __( 'Close' ) }
 					onRequestClose={ toggleModal }
 				>
-					{ shortcutConfig.map( ( config, index ) => (
-						<ShortcutSection key={ index } { ...config } />
-					) ) }
+					<ShortcutSection
+						className="edit-post-keyboard-shortcut-help-modal__main-shortcuts"
+						shortcuts={ mainShortcuts }
+					/>
+					<ShortcutSection
+						title={ __( 'Global shortcuts' ) }
+						shortcuts={ globalShortcuts }
+					/>
+					<ShortcutSection
+						title={ __( 'Selection shortcuts' ) }
+						shortcuts={ [
+							'core/block-editor/select-all',
+							'core/block-editor/unselect',
+						] }
+					/>
+					<ShortcutSection
+						title={ __( 'Block shortcuts' ) }
+						shortcuts={ [
+							'core/block-editor/duplicate',
+							'core/block-editor/remove',
+							'core/block-editor/insert-before',
+							'core/block-editor/insert-after',
+							{
+								keyCombination: '/',
+								description: __( 'Change the block type after adding a new paragraph.' ),
+								/* translators: The forward-slash character. e.g. '/'. */
+								ariaLabel: __( 'Forward-slash' ),
```

Comment:
Should this be an option in the shortcut registration? I see there's a few more usages of this in `globalShortcuts` as well.

Reply:
Potentially yes, we'll probably find a few other options once we refactor all existing shortcuts.